src/web/theme/pages/TryIt/EnhanceTryIt.js

```diff
@@ -0,0 +1,44 @@
+import { compose, withState, withHandlers } from "recompose";
+
+const EnhanceTryIt = steps => {
+  const getActiveStep = (currentStep, steps) => {
+    if (currentStep !== undefined) {
+      return currentStep;
+    } else {
+      return steps[0];
+    }
+  };
+  return compose(
+    withState("tryItState", "setTryItState", {}),
+    withState("currentStep", "setCurrentStep", props =>
+      getActiveStep(props.currentStep, steps)
```

Comment:
If the function is only used here you can simplify your code by returning `props.currentStep` if exist or `steps[0]`